firmware/shared/src/app/app_utils.h

```diff
@@ -11,6 +11,7 @@
 #define MIN3(x, y, z) (MIN(MIN((x), (y)), (z)))
 #define CLAMP(x, min, max) (MAX(MIN(x, max), min))
 #define CLAMP_TO_ONE(x) (((x) <= 0) ? 1 : ((x) > 1 ? 1 : (x))) // initialize to 1 if value is <=0
+#define SQUARE(x) ((x) * (x))
```

Comment:
damn

Reply:
lol faster than pow, nicer code than just doing it straight on the line